web/css/jquery-ui-override.css

```diff
@@ -1,6 +1,10 @@
 /*
  * Default jquery-ui button
  */
+.ui-button-text {
+  font-family: "Segoe UI", "Arial", sans-serif;
```

Comment:
It looks like the original "Segoe UI" declaration is still there, it's just overridden by [these two lines](https://github.com/nasa-gibs/worldview/blob/module-loaders/web/css/sidebar.css#L45-L46) added in https://github.com/nasa-gibs/worldview/commit/8345cb170710cbe9d5dfaa07358a8cdbb7a18c55 that we could probably remove to be more efficient.